### DISCUSSION TITLE
how to apply the creator royalty?

### DISCUSSION THREAD

**Discussion by kenseok** [hashlips-lab/nft-erc721-collection#431] [Q&A]
Hello, I have a question.
How can I set the creator fee?

**Comment by sal-420** [comment]
You would set it on the market place see [https://www.youtube.com/watch?v=JU_fGeK4Wow](url)

**Comment by web3chick** [comment]
> Hello, I have a question. How can I set the creator fee?

Use this version of it. It has been implemented here. 

(https://github.com/datboi-1337/nft-erc721-collection)
